x-pack/plugins/triggers_actions_ui/public/plugin.ts

```diff
@@ -57,7 +65,10 @@ export class Plugin implements CorePlugin<TriggersAndActionsUIPublicPluginSetup,
     };
   }
 
-  public start(core: CoreStart, plugins: PluginsStart) {
+  public start(
+    core: CoreStart,
+    plugins: PluginsStart
+  ): TriggersAndActionsUIPublicPluginStart | undefined {
```

Comment:
Same question from @pmuellr in another PR -> https://github.com/elastic/kibana/pull/58514#discussion_r388271463. Looks like we return early but should still expose the registries.